scripts/clone_subsystems.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
-# Remove it after package manager implementation
+# Remove this script after package manager implementation
 
-cd ..
+APP_ROOT_PATH=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && cd .. && pwd)
+PLATFORM_PATH="${APP_ROOT_PATH}/ostis-web-platform"
+WORKING_PATH=$(pwd)
+
+cd ${APP_ROOT_PATH}
```

Comment:
```suggestion
cd "${APP_ROOT_PATH}"
```
Should be wrapped in quotes everywhere.